File: logica-programacao/javascript/013.js
/*
Dado um número inteiro n:

Gere 1 se for múltiplo de 3

Gere 0 caso contrário
*/ 

let numeroInteiro = Number(prompt("Digite um número inteiro: "));

if (numeroInteiro % 3 === 0) {
    console.log("1");
} else {
    console.log("0");
}

File: logica-programacao/python/013.py
"""
Dado um número inteiro n:

Gere 1 se for múltiplo de 3

Gere 0 caso contrário

"""

numero_inteiro = int(input("Digite um número inteiro: "))

if numero_inteiro % 3 == 0:
    print("1")
else:
    print("0")